src/sx/src/__tests__/__snapshots__/renderPageWithSX.test.js.snap

```diff
@@ -23,23 +23,23 @@ exports[`matches expected output: @supports.json 1`] = `
 
 ~~~~~~~~~~ OUTPUT ~~~~~~~~~~
 @supports (display: grid) {
-  ._3GKV06 {
+  ._3GKV06._3GKV06 {
```

Comment:
Is this correct? Why was this implemented only for `@media` queries? 🤔

Reply:
Yeah, shouldn't be necessary here 🙈

Reply:
Shouldn't? Cannot there be a case where it's needed just like with media queries?

Reply:
Ah, yes you are right, I was not thinking it through. You could have `display:flex`, but if browser supports `grid`, you want `display:grid`, and it would need higher specificity 👍

Reply:
I am also wondering

```
const styles = sx.create({
  text: {
    color: 'pink',
    ':hover': {color: 'cyan'},
    '@media(min-width:900px)': {color: 'blue'}
  }
})
```
in this case, color pink has specificity (0,0,1,0), and hover and media have (0,0,2,0), so the one defined last in the stylesheet will win, meaning it is not certain that it gets color cyan on hover.